backend/lib/services/terminals/terminalBootstrap.js

```diff
@@ -772,6 +727,19 @@ class Bootstrapper extends Queue {
     this.push(handler)
   }
 
+  async handleDependentShoots (seedName) {
+    const query = {
+      fieldSelector: `spec.seedName=${seedName}`
+    }
+    const { items } = await dashboardClient['core.gardener.cloud'].shoots.listAllNamespaces(query)
+    logger.debug(`Bootstrap required for ${items.length} shoots due to terminal bootstrap revision change`)
+    _.forEach(items, shoot => {
+      shoot.kind = 'Shoot' // patch missing kind
+      this.bootstrapState.setBootstrapRequired(shoot)
+      this.bootstrapResource(shoot)
+    })
```

Comment:
No benefit from using lodash
```suggestion
    for (const shoot of items ) {
      shoot.kind = 'Shoot' // patch missing kind
      this.bootstrapState.setBootstrapRequired(shoot)
      this.bootstrapResource(shoot)
    }
```

Reply:
true ✅